moabb/evaluations/base.py

```diff
@@ -216,26 +209,48 @@ def process(self, pipelines, param_grid=None, postprocess_pipeline=None):
             if not (isinstance(pipeline, BaseEstimator)):
                 raise (ValueError("pipelines must only contains Pipelines " "instance"))
 
-        res_per_db = []
-        for dataset in self.datasets:
-            log.info("Processing dataset: {}".format(dataset.code))
-            process_pipeline = self.paradigm.make_process_pipelines(
-                dataset,
-                return_epochs=self.return_epochs,
-                return_raws=self.return_raws,
-                postprocess_pipeline=postprocess_pipeline,
-            )[0]
-            # (we only keep the pipeline for the first frequency band, better ideas?)
-
-            results = self.evaluate(
+        # Prepare dataset processing parameters
+        processing_params = [
+            (
                 dataset,
-                pipelines,
-                param_grid=param_grid,
-                process_pipeline=process_pipeline,
-                postprocess_pipeline=postprocess_pipeline,
+                self.paradigm.make_process_pipelines(
+                    dataset,
+                    return_epochs=self.return_epochs,
+                    return_raws=self.return_raws,
+                    postprocess_pipeline=postprocess_pipeline,
+                )[0],
             )
+            for dataset in self.datasets
+        ]
+        # Parallel processing...
+        parallel_results = Parallel(
+            n_jobs=self.n_jobs, return_as="generator", verbose=10, backend="loky"
+        )(
```

Comment:
Why do you hard code the backend? This should be left as default (`loky`), so it can be changed with `parallel_config` externally. Also, I would add a param controlling verbose or leave it to default too.
```suggestion
        parallel_results = Parallel(
            n_jobs=self.n_jobs, return_as="generator",
        )(
```